src/geographic_supplemental_attribute.jl

```diff
@@ -0,0 +1,33 @@
+"""
+Base type for structs that store infos
+
+Required interface functions for subtypes:
+
+  - get_internal()
+  - get_components_uuid()
+  - get_time_series_container()
+
+Subtypes may contain time series, if no time series container is implemented return nothing
+"""
+struct InfrastructureSystemsGeo <: InfrastructureSystemsSupplementalAttribute
+    geo_json::Dict{String, Any}
```

Comment:
I realize that GeoJSON is a standard, but what we store here will not be JSON. Is there a better term?

Reply:
I was thinking that for editing/serialize/deserialize easy we should just store the dict representation of the JSON but I can be persuaded.